.github/workflows/ruby.yml

```diff
@@ -0,0 +1,59 @@
+name: Ruby
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test_and_coverage:
+
+    runs-on: ubuntu-latest
+    name: Ruby Test + Coverage
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 3.0
+        bundler-cache: true # Runs bundle install and caches gems. See the ruby_test.yml
+                            # example if you need more control over bundler.
+    - name: Test & publish code coverage
+      uses: paambaati/codeclimate-action@v2.7.5
+      env:
+        CC_TEST_REPORTER_ID:  ${{ secrets.CC_TEST_REPORTER_ID || '8ab4bd7f459dfad8d238b1f8b42b8c6d73714ac5746d3e98c81d3e201869fe3b' }}
```

Comment:
What's this about?

Reply:
This is used to gather code coverage statistic over on code climate. The id is provided inline, rather than just as a secret, as secrets aren't available to pull requests. Exposing it is pretty low risk, as all it would let a person do is push up fake coverage statistics.

We still let CC_TEST_REPORTER_ID be specified as a secret in case you want to set up code climate for your private repo. (I use it occasionally when doing particularly large refactors)